integration-test/src/main/java/com/sequenceiq/it/cloudbreak/action/sdx/SdxSetFlowChainIdAction.java

```diff
@@ -11,7 +11,7 @@
 
 public class SdxSetFlowChainIdAction implements Action<SdxTestDto, SdxClient> {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(SdxUpgradeAction.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(SdxOsUpgradeAction.class);
```

Comment:
Please use the `SdxSetFlowChainIdAction` class for the logger initialization.

Reply:
Ok, thanks!